recipes/pydtmc/meta.yaml

```
@@ -0,0 +1,52 @@
+package:
+  name: pydtmc
+  version: "6.2.0"
+
+source:
+  url: https://pypi.io/packages/source/P/PyDTMC/PyDTMC-6.2.0.tar.gz
```

Would be better to use version as a template variable:

```suggestion
{% set version = "6.2.0" %}

package:
  name: pydtmc
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/P/PyDTMC/PyDTMC-{{ version }}.tar.gz
```